Lua/LuaBindingsEntities.cpp

```diff
@@ -598,6 +598,30 @@ namespace RTE {
 		.def("ResetEmissionTimers", &Emission::ResetEmissionTimers);
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	LuaBindingRegisterFunctionDefinitionForType(EntityLuaBindings, Gib) {
+		return luabind::class_<Gib>("Gib")
+
+		.property("ParticlePreset", &Gib::GetParticlePreset, &Gib::SetParticlePreset)
+		.property("MinVelocity", &Gib::GetMinVelocity, &Gib::SetMinVelocity)
+		.property("MaxVelocity", &Gib::GetMaxVelocity, &Gib::SetMaxVelocity)
+		.property("SpreadMode", &Gib::GetSpreadMode, &Gib::SetSpreadMode)
+
+		.def_readwrite("Offset", &Gib::m_Offset)
```

Comment:
Why are you using def_readwrite here instead of properties? Were you too lazy to make a getter and setter?

Reply:
We have enough pointless getters/setters just for the sake of lua, thank you.

Reply:
lol sure, you lazy basterd :P